skbuild/resources/cmake/FindF2PY.cmake

```diff
@@ -0,0 +1,98 @@
+# stand-alone executables.
+#
+# The following variables are defined:
+# ::
+#
+#   F2PY_EXECUTABLE                     - absolute path to the F2PY executable
+#                                         
+#   F2PY_VERSION                        - the F2PY version
+#
+#   F2PY_VERSION_MAJOR                  - the F2PY major version
+#
+#   F2PY_VERSION_MINOR                  - the F2PY minor version
+#
+#   F2PY_VERSION_MICRO                  - the F2PY micro version
+#
+#
+# Find the F2PY program, which comes as part of NumPy.
+#
+#
+#=============================================================================
+# Copyright 2011-2017, the PyNE Development Team. All rights reserved.
+# 
+# Redistribution and use in source and binary forms, with or without modification, are
+# permitted provided that the following conditions are met:
+# 
+#    1. Redistributions of source code must retain the above copyright notice, this list of
+#       conditions and the following disclaimer.
+# 
+#    2. Redistributions in binary form must reproduce the above copyright notice, this list
+#       of conditions and the following disclaimer in the documentation and/or other materials
+#       provided with the distribution.
+# 
+# THIS SOFTWARE IS PROVIDED BY THE PYNE DEVELOPMENT TEAM ``AS IS'' AND ANY EXPRESS OR IMPLIED
+# WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+# FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL <COPYRIGHT HOLDER> OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+# ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+# NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+# ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+# 
+# The views and conclusions contained in the software and documentation are those of the
+# authors and should not be interpreted as representing official policies, either expressed
+# or implied, of the stakeholders of the PyNE project or the employers of PyNE developers.
+#=============================================================================
+
+find_package(NumPy)
+
+find_program(F2PY_EXECUTABLE F2PY ${DEPS_BIN_HINTS})
+if(NOT F2PY_EXECUTABLE)
+  if(${PYTHON_VERSION_MAJOR} GREATER 2)
+    find_program(F2PY_EXECUTABLE3 F2PY3 ${DEPS_BIN_HINTS})
+    if(F2PY_EXECUTABLE3)
+      set(F2PY_EXECUTABLE ${F2PY_EXECUTABLE3})
+    endif(F2PY_EXECUTABLE3)
+  elseif(${PYTHON_VERSION_MAJOR} LESS 3)
+    # because arch is dumb
+    find_program(F2PY_EXECUTABLE2 F2PY2 ${DEPS_BIN_HINTS})
+    if(F2PY_EXECUTABLE2)
+      set(F2PY_EXECUTABLE ${F2PY_EXECUTABLE2})
+    endif(F2PY_EXECUTABLE2)
+  endif(${PYTHON_VERSION_MAJOR} GREATER 2)
+endif(NOT F2PY_EXECUTABLE)
```

Comment:
Consider running this script to remove block end arguments:

https://github.com/jcfr/cmake-auto-style/blob/master/docs/cmake-commands-remove-block-end-arguments.md#cmake-commands-remove-block-end-arguments